Automattic/vip-go-mu-plugins
# Updated ElasticPress to latest Automattic/ElasticPress

## Description

Updated ElasticPress to Automattic/ElasticPress#acb044b246efa6f85d3de58c03e8abfea0ac5572
## Changelog Description

### Plugin Updated: VIP Search

Updated to the latest version of VIP Search. The only change is bringing `search/elasticpress/includes/classes/Indexable/Post/Post.php` in line with our coding standards.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Apply PR.
2. Perform various VIP Search actions(searches, indexing, etc...)
3. There should be no new errors.
